cli/src/command.rs

```diff
@@ -293,6 +304,31 @@ where
 pub fn run() -> Result<()> {
 	let cli: Cli = Cli::from_args();
 
+	#[cfg(feature = "pyroscope")]
+	let mut pyroscope_agent_maybe = if let Some(ref agent_addr) = cli.run.pyroscope_server {
+		let address = agent_addr
+			.to_socket_addrs()
+			.map_err(Error::AddressResolutionFailure)?
+			.next()
+			.ok_or_else(|| Error::AddressResolutionMissing)?;
+		// The pyroscope agent requires a `http://` prefix, so we just do that.
+		let mut agent = pyro::PyroscopeAgent::builder(
+			"http://".to_owned() + address.to_string().as_str(),
+			"polkadot".to_owned(),
+		)
+		.sample_rate(100)
```

Comment:
FWIW, flamegraph creator recommends[^1]
> The odd numbered rates, 99 and 199, are used to avoid sampling in lockstep with other activity and producing misleading results.

And not-perf uses 900 by default[^2]. @koute any insights on this? 

[^1]: https://www.brendangregg.com/FlameGraphs/cpuflamegraphs.html
[^2]: https://cs.github.com/koute/not-perf/blob/e75188b1067e11c692ed172dc1333a6a031ef55c/src/args.rs?q=900#L170

Reply:
Made the sample rate a prime number above 100, to retain accuracy and not introduce too much overhead. This is just an initial number, we'll have to adjust this based on practical experience, hence I'd like to keep this on the lower, less invasive side for the time being.

Reply:
Yeah, I think it's a good idea to pick a "less round" sampling rate.

For `not-perf` there wasn't really any deep meaning behind why I picked 900, besides the fact that I wanted it high to make the results more detailed/reliable.